src/components/molecules/card/AnyWayCard.tsx

```diff
@@ -83,9 +279,9 @@ const AnyWayCard: FC<IProps> = ({
 }) => {
   const [element, setElement] = useState({});
   const [isOpen, setOpen] = useState(false);
-  const [widgateOpen,SetWidgetOpen] = useState('')
+  const [widgateOpen, SetWidgetOpen] = useState('')
 
-  const handleCardEditorOpen = (name:string) =>{ 
+  const handleCardEditorOpen = (name: string) => {
```

Comment:
All these space changes are things that happen automatically sometimes by VScode, but you usually don't want to push them because, even though they are good changes, are not related to the issue